src/documents/ModelRelay.md

```diff
@@ -129,10 +180,42 @@ that.child.applier.change("volume", 1);
 console.log(that.model.volume);       // 100 - inverse transformed to accept update from child component
 ```
 
-In general those transformations which are _**invertible**_ are the best choice for this kind of linkage. If the transforms are not invertible (or have no inverse defined in the framework), the updates will propagate only in one direction. This can still be highly useful.
+In general those transformations which are _**invertible**_ are the best choice for this kind of linkage. If the transforms are not invertible (or have no inverse defined in the framework), 
+the updates will propagate only in one direction. This can still be highly useful. In addition to its invertibility, the propagation of updates through a relay rule can be fine-tuned by the options
+`forward` and `backward`:
+
+####Controlling Propagation Through a Relay Rule####
+
+Each explicit relay rule can accept options `forward` and `backward` which allows the configurer to control the occasions on which the relay is operated in those directions &#8212; that is, `forward` representing
+the direction from `source` to `target`, and `backward` representing the direction from `target` to `source`. If the transform is not
+invertible, the `backward` option is ignored, and the system will behave as if it read `backward: "never"`. The control which can be exerted is over whether the relay is operated either during the
+<a href="#the-initial-transaction">initial transaction</a>, the "mature life" of the component after construction, or neither, or both. The default value for these options is `"always"` which
+implies that the system will always operate the relay rule if it can. 
+
+Restricting the relay to `liveOnly` in one direction can often be useful to express the fact that whatever default value is 
+configured into the model at the other end of the relay is to be ignored. This is frequently useful, since default model values typically propagate outwards from an implementation core and
+are normally expected to overwrite defaults which are held in more reusable components held at the periphery. 
+
+A rule of `never` can be useful to express the fact that a particular component
+is only expected to act as a source of updates and is not prepared to act on them from the outside &#8212; it may control some kind of device, for example, which by construction cannot accept input.
+
+A rule of `initOnly` is less often useful, but can be helpful in controlling the process whereby initial default values of more abstract kinds of models are computed.
+
+Compare these directives with the related ones used for source guarding in [model listeners](ChangeApplierAPI.md#source-tracking-and-filtering-in-model-listener-blocks) (`init`, `relay` and `local`).
```

Comment:
This sentence might be more useful if you actually said something about the comparison.